site/components/src/main/java/uk/nhs/digital/common/components/catalogue/ApiCatalogueHubComponent.java

```diff
@@ -64,15 +64,16 @@ public void doBeforeRender(final HstRequest request, final HstResponse response)
 
         long endTime = System.currentTimeMillis();
         long duration = endTime - startTime;
-        log.info("End of method: doBeforeRender in ApiCatalogueHubComponent  at " + endTime + " ms. Duration: " + duration + " ms");
+        log.info("End of method: doBeforeRender in ApiCatalogueHubComponent  at {} ms. Duration: {} ms", endTime, duration);
     }
 
     @Override
-    protected <T extends EssentialsListComponentInfo> Pageable<HippoBean> doFacetedSearch(HstRequest request, T paramInfo, HippoBean scope) {
-
+    public Pageable<HippoBean> doFacetedSearch(HstRequest request, EssentialsListComponentInfo paramInfo, HippoBean scope) {
```

Comment:
What's the reason for this change? Leaving it as it was, `ApiCatalogueHubComponentTest` will continue to compile because it is already package and subclass visible. 

Moreover, we lose flexibility if we need to subclasses or extend its behaviour in the future. The method no longer accepts a parameter of any subclass of `EssentialsListComponentInfo`. Instead, it now strictly expects an `EssentialsListComponentInfo`.

Reply:
Also, I am not convinced this modified method (without the generic type parameter) would be considered an override anymore.

Reply:
Reverted the code as like master.